tests/unit/cli/builder/test_cli_builder.py

```diff
@@ -0,0 +1,420 @@
+"""Unit-Test Case for otx.cli.builder.builder."""
```

Comment:
Just to follow the same directory structure, would it be an idea to place this test into `tests/unit/otx/cli/builder/test_cli_builder.py`?

Reply:
Or do you think all of the unit tests are already part of `otx` directory, hence we don't need to create `tests/unit/otx`?

Reply:
Your second comment is correct.
This doesn't matter, but since the target of tests is the `otx` folder (since other folders are not in the target), there seems to be no need to put `otx` in `tests/unit`.

Reply:
Thanks for the clarification!